tests/ert/ui_tests/gui/test_main_window.py

```diff
@@ -854,3 +856,84 @@ def wait_for_simulation_completed():
         choice.trigger()
         find_and_check_selected("button_Start_simulation", False)
         find_and_check_selected("button_Simulation_status", True)
+
+
+from pytestqt.qtbot import QtBot
+
+
+def test_that_invalid_experiments_are_disabled(opened_main_window_poly, qtbot: QtBot):
+    gui: ErtMainWindow = opened_main_window_poly
+
+    def find_and_click_button(button_name: str):
+        button = gui.findChild(QToolButton, button_name)
+        assert button
+        qtbot.mouseClick(button, Qt.LeftButton)
+
+    def run_experiment():
+        run_experiment_panel = wait_for_child(gui, qtbot, ExperimentPanel)
+        qtbot.wait_until(lambda: not run_experiment_panel.isHidden(), timeout=5000)
+        assert run_experiment_panel.run_button.isEnabled()
+        qtbot.mouseClick(run_experiment_panel.run_button, Qt.LeftButton)
+
+    def wait_for_simulation_completed():
+        run_dialogs = get_children(gui, RunDialog)
+        dialog = run_dialogs[-1]
+        qtbot.wait_until(lambda: not dialog.isHidden(), timeout=5000)
+        qtbot.wait_until(lambda: dialog.is_simulation_done() == True, timeout=15000)
+
+    def check_manage_experiment_tab(
+        expected_number_of_valid_experiments: int,
+        expected_number_of_invalid_experiments: int,
+    ) -> None:
+        find_and_click_button("button_Manage_experiments")
+        experiments_panel = wait_for_child(gui, qtbot, ManageExperimentsPanel)
+
+        storage_widget = get_child(experiments_panel, StorageWidget)
+        tree_view = get_child(storage_widget, QTreeView)
+        tree_view_model: StorageModel = tree_view.model()
+
+        number_of_found_invalid_experiments = 0
+        number_of_found_valid_experiments = 0
+        for row in range(tree_view_model.rowCount()):
+            index = tree_view_model.index(row, 0)
+            entry_is_enabled = bool(
+                tree_view_model.flags(index) & Qt.ItemIsEnabled
+            )  # here we check if it is enabled
+            if entry_is_enabled:
+                number_of_found_valid_experiments += 1
+            else:
+                number_of_found_invalid_experiments += 1
+                assert (
+                    tree_view_model.data(index, Qt.ItemDataRole.ToolTipRole)
+                    == "Responses file is missing"
+                )
+        assert (
+            number_of_found_invalid_experiments
+            == expected_number_of_invalid_experiments
+        )
+        assert number_of_found_valid_experiments == expected_number_of_valid_experiments
+
+    def check_plot_tool(expected_number_of_cases: int) -> None:
+        find_and_click_button("button_Create_plot")
+        # Due to the fact that we create new instances of PlotWindow on tab change, QtBot is defaulting to the first child
```

Comment:
If you refer to this;
```
            if index_name == "Create plot":
                if self._plot_window:
                    self._plot_window.close()
                self._plot_window = PlotWindow(self.config_file, self)
```
It should be possible to check if the PlotWindow is open or not. Also, the window should be accessible via;
```
gui._plot_window
```